### ISSUE TITLE
Consistent naming and documentation of attributes of nodes/vertices

### ISSUE THREAD

**Issue by Entenwilli** [DataFlowAnalysis/DataFlowAnalysis#178]
Currently, the naming of attributes of nodes/vertices is not consistent. As discussed in the meeting today, the following terms should be used:
- `Node Label` Previously also called `Vertex Characteristics` or `Node Characteristics`
- `Data Label` Previously called `Data Characteristics`
- `Vertex` Previously also called `Node`

In addition to renaming occurrences in code and documentation, we should also add central documentation for the chosen naming and provide context in regard to the PCM and DFD models

**Comment by Nicolas-Boltz** [comment]
Is this still an issue?

**Comment by Entenwilli** [comment]
Yes, some documentation and variable names are still inconsistent
